mindmeld/tokenizer.py

```diff
@@ -29,21 +29,52 @@ class Tokenizer:
     """The Tokenizer class encapsulates all the functionality for normalizing and tokenizing a
     given piece of text."""
 
-    _ASCII_CUTOFF = ord("\u0080")
-
     def __init__(self, app_path=None, exclude_from_norm=None):
         """Initializes the tokenizer.
 
         Args:
             exclude_from_norm (optional) - list of chars to exclude from normalization
         """
 
-        self.ascii_folding_table = self.load_ascii_folding_table()
         self.exclude_from_norm = exclude_from_norm or []
         self.config = get_tokenizer_config(app_path, self.exclude_from_norm)
+        self.language, _ = get_language_config(app_path)
+        self._tokenizer = self._get_tokenizer()
+        self._normalizer = self._get_normalizer()
         self._custom = False
         self._init_regex()
 
+    def _get_tokenizer(self):
+        """Gets the 'tokenizer' specified in the config.
+
+        Returns:
+            tokenizer (Tokenizer): Tokenizer specified in the config.
+        """
+        tokenizer = self.config.get("tokenizer")
+        if not tokenizer:
+            raise AssertionError("'tokenizer' cannot be None.")
+        elif tokenizer == NoOpTokenizer.__name__:
```

Comment:
Why do we do this?

Reply:
Just to check if "NoOpTokenizer" is being passed in. Should I skip this check instead?

Reply:
Removed